stable/kibana/README.md

```diff
@@ -36,38 +36,48 @@ The command removes all the Kubernetes components associated with the chart and
+| `ingress.enabled`                             | Enables Ingress                            | `false`                                |
+| `ingress.annotations`                         | Ingress annotations                        | None:                                  |
+| `ingress.hosts`                               | Ingress accepted hostnames                 | None:                                  |
+| `ingress.tls`                                 | Ingress TLS configuration                  | None:                                  |
+| `nodeSelector`                                | node labels for pod assignment             | `{}`                                   |
+| `podAnnotations`                              | annotations to add to each pod             | `{}`                                   |
+| `replicaCount`                                | desired number of pods                     | `1`                                    |
+| `revisionHistoryLimit`                        | revisionHistoryLimit                       | `3`                                    |
+| `serviceAccountName`                          | serviceAccount that will run the pod       | `nil`                                  |
+| `resources`                                   | pod resource requests & limits             | `{}`                                   |
+| `priorityClassName`                           | priorityClassName                          | `nil`                                  |
+| `service.externalPort`                        | external port for the service              | `443`                                  |
+| `service.internalPort`                        | internal port for the service              | `4180`                                 |
+| `service.externalIPs`                         | external IP addresses                      | None:                                  |
+| `service.loadBalancerIP`                      | Load Balancer IP address                   | None:                                  |
+| `service.nodePort`                            | NodePort value if service.type is NodePort | None:                                  |
+| `service.type`                                | type of service                            | `ClusterIP`                            |
+| `service.annotations`                         | Kubernetes service annotations             | None:                                  |
+| `service.labels`                              | Kubernetes service labels                  | None:                                  |
+| `tolerations`                                 | List of node taints to tolerate            | `[]`                                   |
+| `ConfigmapDashboardImport.enabled`            | Enable Dashboard import                    | `false`                                |
```

Comment:
Seems superfluous to have `Configmap` in this variable and helm is defaulting to using camelCase. Can we rename this section `dashboardImport` rather than `ConfigmapDashboardImport` ...?